cleanlab_studio/studio/studio.py

```diff
@@ -345,7 +345,7 @@ def download_embeddings(
         return np.asarray(api.download_array(self._api_key, cleanset_id, "embeddings"))
 
     def TLM(
-        self, *, quality_preset: trustworthy_language_model.QualityPreset = "low"
+        self, *, quality_preset: trustworthy_language_model.QualityPreset = "medium"
     ) -> trustworthy_language_model.TLM:
```

Comment:
dont forget to update this line here:

https://help.cleanlab.ai/tutorials/tlm/#advanced-usage

`The default preset is `

I'd try to always open the PRs for such updates simultaneously across the repos (just mark the docs PR as draft until this one is merged), otherwise we will forget to update it